doc/stdenv.xml

```diff
@@ -2657,6 +2657,14 @@ addEnvHooks "$hostOffset" myBashFunction
        When a build failure happens there will be an instruction printed that
        shows how to attach with <literal>cntr</literal> to the build sandbox.
       </para>
+      <para>
+       <emphasis>Note:</emphasis> this won't work with remote builds as the
```

Comment:
Why not use `<note></note>`?

Reply:
Admittedly I didn't know about this, thanks!